docker: Initial Dockerfile

jcorde opened the pull request:
Create Dockerfile derived from the original Dockerfile for Zephyr
Project's Docker image.

Changes:
	Remove VNC support.
 	Cloning Zephyr and KNoT used git repositories.
	Always using CESAR's Zephyr Project git fork.
	Install Zephyr-KNoT-SDK requirements.
	Allow user to use desired folder as current working directory.
	Install Segger Jlink drivers from an external file.

The Segger Jlink drivers path has to be provided during compilation.

Add LICENSE.txt file as the original project is protected by Apache
License.

Closes: #27

Signed-off-by: jvccCESAR <jvcc@cesar.org.br>

jcorde committed:
docker: Initial Dockerfile

Create Dockerfile derived from the original Dockerfile for Zephyr
Project's Docker image.

Changes:
	Remove VNC support.
 	Cloning Zephyr and KNoT used git repositories.
	Always using CESAR's Zephyr Project git fork.
	Install Zephyr-KNoT-SDK requirements.
	Allow user to use desired folder as current working directory.
	Install Segger Jlink drivers from an external file.

The Segger Jlink drivers path has to be provided during compilation.

Add LICENSE.txt file as the original project is protected by Apache
License.

Closes: #27

Signed-off-by: jvccCESAR <jvcc@cesar.org.br>

ramonhpr reviewed (commented):
The patch is good. Just add a README on how to build and run it.

vitbaq reviewed (commented):
I agree with Ramon

jcorde committed:
docker: Add README file

Add README.md file with instructions on how to build and use the docker
container.

Signed-off-by: Joao Cordeiro <jvcc@cesar.org.br>

jcorde commented:
Hi, @vitbaq and @ramonhpr . I just added the README file to https://github.com/CESARBR/zephyr-knot-sdk/pull/31/commits/31a484648584495d35ff0d3d71265ab54bc5e204

vitbaq reviewed (changes requested):
Makes it clear that inside docker the user can use cli functions and link with cli guide.
And illustrate when the user is in the docker container terminal.

jcorde commented:
@vitbaq changes made to README file
